### ISSUE TITLE
data input label number doesnt  match network output label

### ISSUE THREAD

**Issue by lucaskyle** [haofanwang/accurate-head-pose#7]
hey there：

I have a problem here，
in your data_set:

        bins_2 = np.array(range(-99, 102, 66))
        binned_pose_2 = np.digitize([yaw, pitch, roll], bins_2) - 1

that means   -99 to 99 will be divided into  3 parts, 3 labels.
But your network output labels_2 is a 6-label output... 

       self.fc_pitch_3 = nn.Linear(512 * block.expansion, 6)
that means the labels should be divided into 6 labels 6 classes(in the paper.)

**Comment by lucaskyle** [comment]
in paper:
 198 classes, 66 classes, 18 classes, 6 classes and 2 classes.

it should be:


        bins = np.array(range(-99, 102, 1))
        binned_pose = np.digitize([yaw, pitch, roll], bins) - 1
        
        bins_0 = np.array(range(-99, 102, 3))
        binned_pose_0 = np.digitize([yaw, pitch, roll], bins_0) - 1
        
        bins_1 = np.array(range(-99, 102, 11))
        binned_pose_1 = np.digitize([yaw, pitch, roll], bins_1) - 1
        
        bins_2 = np.array(range(-99, 102, 33))
        binned_pose_2 = np.digitize([yaw, pitch, roll], bins_2) - 1
        
        bins_3 = np.array(range(-99, 102, 99))
        binned_pose_3 = np.digitize([yaw, pitch, roll], bins_3) - 1

**Comment by haofanwang** [comment]
Hi, @lucaskyle, thanks for following our work.  You're right, they should be consistent. You can PR and I will merge it later.

**Comment by lucaskyle** [comment]
兄弟 搞完了  记得给好评...
